Epam.TestAutomationProject/Epam.TestAutomation.Tests/CareersPageTests.cs

```diff
@@ -0,0 +1,44 @@
+﻿using Epam.TestAutomation.Core.Browser;
+using Epam.TestAutomation.Core.Elements;
+using Epam.TestAutomation.Pages.Pages;
+using OpenQA.Selenium.Interactions;
+
+
+namespace Epam.TestAutomation.Tests
+{
+    [TestFixture]
+    public class CareersPageTests : BaseTest
+    {
+        private CareersPage _careersPage;
+        private JobListingsPage _jobListingsPage;
+        private MainPage _mainPage;
+        private Actions _actions;
+
+        [SetUp]
+        public void SetUp()
+        {
+            _mainPage = new MainPage();
+            _careersPage = new CareersPage();
+            _jobListingsPage = new JobListingsPage();
+            _actions = new Actions(Browser.Instance.Driver);
```

Comment:
You already have the instance of the Actions class in the Browser class son reuse from Browser.